CIP-0098/README.md

```diff
@@ -0,0 +1,474 @@
+---
+CIP: 98?
+Title: DAT Metadata Standard
+Category: Metadata
```

Comment:
```suggestion
Category: Tokens
```
For me the category of `Tokens` fits this proposal a little better, as it relates to "About tokens (fungible or non-fungible) and minting policies in general." see [CIP-01 Categories](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#categories).

Reply:
Makes sense!